cli/src/main.rs

```diff
@@ -28,52 +31,87 @@ struct Args {
     // Config file
     #[arg(short, long)]
     config_file: Option<PathBuf>,
+
+    #[arg(short, long, default_value = "false")]
+    stdin: bool,
+
+    #[arg(short, long, default_value = "false", default_value_if("stdin", ArgPredicate::IsPresent, "true"))]
+    quiet: bool,
 }
 
 fn main() {
     let args = Args::parse();
 
     let settings = create_settings(&args).unwrap();
+    let quiet = args.quiet;
 
     // Print settings
-    println!("{}", toml::to_string_pretty(&settings).unwrap());
-
-    let is_dir = fs::metadata(&args.input_pattern)
-        .map(|meta| meta.is_dir())
-        .unwrap_or(false);
+    if !quiet {
+        println!("{}", toml::to_string_pretty(&settings).unwrap());
+    }
 
-    let glob_pattern = if is_dir {
-        format!("{}/**/*.rs", &args.input_pattern)
+    if args.stdin {
+        match format_stdin_result(settings) {
```

Comment:
Will `if let Err()` be more clean maybe(?)

Reply:
Done